### PR TITLE
Add attendees to meeting

### LINKED ISSUE #5: Add all people in the channel to the meeting
Meeting creation allows you to add the people to the meeting directly. This will make it so you don't have to give permission to enter the meeting to each one.

The idea of this ticket is to add all people signed in into MSTeams Calling that are on the channel to the meeting.

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Add attendees to meeting, and some refactoring.

#### Ticket Link
Fix #5

### CODE REVIEW

**Review by jfrerich — COMMENTED** [review]
LGTM.  At first I thought the following was a bit intrusive, but I guess this is how we do zoom meetings. I was concerned about sending an invite to an entire channel, especially with a large number of users in the channel.  Wonder if we should add some setting to zoom/msteams that would allow setting the following:

* warn and ask for confirm when channel has X number fo users (allow configuring X number of users setting)

> The idea of this ticket is to add all people signed in into MSTeams Calling that are on the channel to the meeting.

File: server/post.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+			if err != nil {
+				continue
+			}
+			attendees = append(attendees, attendeeInfo)
+		}
+	}
+
+	client := remote.NewClient(conf, userInfo.OAuthToken, p.API)
+
+	meeting, err := client.CreateMeeting(userInfo, attendees...)
```
Comment (iomodo) [inline server/post.go:51]: just curious why did you use variadic func?

**Review by iomodo — APPROVED** [review]
LGTM as well. @jfrerich's concern is legit though.

**Comment by larkox** [comment]
@jfrerich I see what you mean. This is something that has come around for Zoom before too. The "quick solution" was to just allow the user to delete its own meeting, in case if by mistake it did create one in a big channel. I will create a ticket on Zoom and raise this situation with Aaron.

File: server/post.go (hunk 1 above)
Comment (larkox) [reply server/post.go:51]: I guess I did originally to keep any `CreateMeeting(creator)` keep working after adding the changes, but I see that I only use this with a list of attendees. That being said, what do you think is best?

**Review by jfrerich — APPROVED** [review]
LGTM, but needs a conflict fix before passing to QA

**Pull request merged**